templates/contacts/contact_invite_participants.haml

```diff
@@ -0,0 +1,479 @@
+-extends "smartmin/list.html"
+-load smartmin sms contacts temba
+-load i18n humanize
+
+-block page-title
+  -if current_group
+    {{current_group.name}}
+  -else
+    {{title}}
+
+-block page-top
+  {% render as page_top %}
+    {{block.super}}
+  {% endrender %}
+  -if has_contacts
+    {{page_top}}
+
+-block title-icon
+  %span.title-icon
+    .glyph.icon-users
+
+-block above-bar
+  -block top-form
+    %form{method:'get'}
+      %input.input-medium.search-query{'type':'text', 'placeholder':'{% trans "Search" %}', 'name':"search", 'value':"{{search}}"}
+
+-block content
+  #pjax
+    -block pjax
+      .row-fluid
+        .span3.sidebox
+          %div.contacts{style:"margin-bottom: 10px"}
+            - if org_perms.contacts.contact_import
+              %a#id_manage_contacts.btn.btn-primary.btn-block{href:"{% url 'contacts.contact_list' %}"}
+                -trans "Manage Contacts"
+
+          #sidebar-nav
+            %ul.nav.nav-list
+              %li.nav-header
+                -trans "Contacts"
+              - for folder in folders
+                %li{'class':'{% if request.get_full_path == folder.url %}active{% endif %}'}
+                  %a{'href':'{{folder.url}}'}
+                    {{folder.label}} ({{ folder.count | intcomma }})
+
+              %li.nav-header
+                -trans "Groups"
+              - for group in groups
+                %li{'class':'{% if current_group.id == group.pk %}active{% endif %}'}
+                  %a{'href':'{% url "contacts.contact_invite_participants"  %}?group={{group.uuid}}'}
+                    -if group.is_ready
+                      {{group.label}} ({{ group.count | intcomma }})
+                    -else
+                      {{group.label}}
+                      %i.spin.icon-loop
+
+        - if has_contacts
+          - block contacts-list
+            .span9
+
+              .control-group
+                %label.control-label
+                  %trans Opt-in flow
+
+                %form.controls{method:"POST"}
+                  - csrf_token
+                  %select#id_opt_in_flow{float:'left', name:'optin_flow_uuid', data-url-template:"{% url 'flows.flow_broadcast' '0' %}"}
+                    - for flow in flows
+                      %option{value:'{{flow.uuid}}'}
```

Comment:
The only thing that I noticed is that the default Opt-In flow from the organization config field should be selected here by default, and the user should be able to click on _Open in the flow editor_ and go to the flow editor

Reply:
Have added selected attribute and fixed open in flow editor link.